### PR TITLE
Fixes graphql root schema import to be generic (instead of environmen…

### PR DESCRIPTION
**Pull request by ajainarayanan**
…t specific)

~**On-Hold**: Right now it works from intelliJ and in dev environment but not in the SDK we build. @miguelvelezmj25 is working on it.~ Fixed it.

### CODE REVIEW

File: cdap-ui/pom.xml [hunk 1]
```diff
@@ -314,8 +314,8 @@
                     <copy todir = "${stage.opt.dir}/server/config/themes">
                       <fileset dir = "server/config/themes" />
                     </copy>
-                    <copy todir = "${stage.opt.dir}/graphql">
-                      <fileset dir = "graphql" />
+                    <copy todir = "${stage.opt.dir}/graphql/schema">
```
Comment (elfenheart) [inline cdap-ui/pom.xml:317]: won't you need the fragments file as well?

File: cdap-ui/graphql/graphql.js [hunk 2]
```diff
@@ -14,6 +14,7 @@
  * the License.
  */
 
+var path = require('path');
```
Comment (miguelvelezmj25) [inline cdap-ui/graphql/graphql.js]: `var` -> `const`?

File: cdap-ui/pom.xml (hunk 1 above)
Comment (ajainarayanan) [reply cdap-ui/pom.xml:317]: We don't need the fragmentTypes.json in the server right? This gets bundled in the client code in main.js I believe?

**Review by elfenheart — APPROVED** [review]
👍 LGTM

File: cdap-ui/pom.xml (hunk 1 above)
Comment (miguelvelezmj25) [reply cdap-ui/pom.xml:317]: Correct. fragmentTypes.json is used in the main.js file, but there is a path specified: `import introspectionQueryResultData from '../../graphql/fragments/fragmentTypes.json';`. Is this path still correct after your changes on how to copy the graphql directory?

**Comment by ajainarayanan** [comment]
Green build: https://builds.cask.co/browse/CDAP-UDUT372-1

**Pull request merged**